### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by Marta108**

### CODE REVIEW

File: staff/marta-cervera/challenges/calculator-app/calculadora. 2/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## How it works
+The user who wants to use it must click on a number, after clik again on the operation that has to be done and finally the second number. The user must push the button equal to know the result of the operation. Also it has a button, for reset the calculator.
+## Technical Description 
+### What Operations can do?
+- Add
+- Substract
+- Divide
+- Multiply
+## Index.html
+The app structure is rectangle that inside, it has a structure of buttons taht corresponds with numbers and operations. Above the buttos there is a place in which area, the numbers are going to appear as soon as you press the botton.
```
Comment (mariajbellido) [inline staff/marta-cervera/challenges/calculator-app/calculadora. 2/README.md]: small typos: taht -> that / button -> buttons

File: staff/marta-cervera/challenges/calculator-app/calculadora. 2/main.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+var buttons = document.querySelectorAll("button");
+var display = buttons[0];
+
+var actual =" "
+var aux
+var operation
+var deleteNumber
+
+buttons [1].onclick= function() {
+    actual= actual + "7"
```
Comment (mariajbellido) [inline staff/marta-cervera/challenges/calculator-app/calculadora. 2/main.js]: missing ;

File: staff/marta-cervera/challenges/Tic-tac-toe/main.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+var symbol = "X"
+var state = [];
+
+var buttons = document.querySelectorAll ("button");
+
+var button 0 = button [0]
+buttons [0].onclick = function () {
+    state = symbol;
+
```
Comment (MarioRubio81) [inline staff/marta-cervera/challenges/Tic-tac-toe/main.js:11]: Forgotten symbol }

File: staff/marta-cervera/challenges/Tic-tac-toe/style.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+body {
+background: url(tic\ tac\ toe.jpg);
```
Comment (manuelbarzi) [inline staff/marta-cervera/challenges/Tic-tac-toe/style.css]: do never use filenames with empty spaces. rename the file to tic-tac-toe.jpg, instead, and update this reference in the css code.

File: staff/marta-cervera/challenges/Tic-tac-toe/main.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+//declaramos las variables: simbolo( que tiene que ir cambiando según el turno del jugador), estado, y la posicion ganadora
+
+var symbol = "X"
```
Comment (betoDavila86) [inline staff/marta-cervera/challenges/Tic-tac-toe/main.js]: missing semicolon ;

File: staff/marta-cervera/challenges/Tic-tac-toe/style.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+
```
Comment (betoDavila86) [inline staff/marta-cervera/challenges/Tic-tac-toe/style.css:4]: remove empty lines.

File: staff/marta-cervera/challenges/Tic-tac-toe/README.MD [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# TIC TAC TOE
```
Comment (manuelbarzi) [inline staff/marta-cervera/challenges/Tic-tac-toe/README.MD:1]: rename folder to all lowercase characters => tic-tac-toe-app

File: staff/marta-cervera/challenges/Tic-tac-toe/style.css [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    color: tomato;
+    text-align: center;
+}
+
+
+.table {
+    display : grid;
+    grid-template-columns:  20% 20% 20% ;
+    grid-template-rows: 20vh 20vh 20vh ;
+
```
Comment (betoDavila86) [inline staff/marta-cervera/challenges/Tic-tac-toe/style.css:25]: remove empty lines.

File: staff/marta-cervera/challenges/Tic-tac-toe/tic tac toe.js [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+param {string} symbol. The symbol that is playing ("x" or "o")
+param {number} row. The row which corresponds the symbol that is playin
+param {number} colum. The colum which corresponds the symbol that is playing
+
+
+
+*/
+
+var state = []
```
Comment (betoDavila86) [inline staff/marta-cervera/challenges/Tic-tac-toe/tic tac toe.js]: missing semicolon.

File: staff/marta-cervera/challenges/Tic-tac-toe/tic tac toe.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+ // X wins
+
+    else if (table[0] === "O" && table [1]=== "O" && table [2]=== "O" || table[3] === "O" && table [4]=== "O" && table [5]=== "O" ||
+    table [6]=== "O" && table [7]=== "O" && table [8]=== "O")// O wins
+
+}
+
+function Status(){
+
+    var state =[]
```
Comment (betoDavila86) [inline staff/marta-cervera/challenges/Tic-tac-toe/tic tac toe.js]: missing semicolon here.

**Review by betoDavila86 — CHANGES_REQUESTED** [review]

File: staff/marta-cervera/challenges/calculator-app/calculadora. 2/README.md (hunk 1 above)
Comment (Marta108) [reply staff/marta-cervera/challenges/calculator-app/calculadora. 2/README.md]: Thank you Maria Jesús. Resolved

File: staff/marta-cervera/challenges/calculator-app/calculadora. 2/main.js (hunk 2 above)
Comment (Marta108) [reply staff/marta-cervera/challenges/calculator-app/calculadora. 2/main.js]: Thank you again

**Comment by simonbesteiro** [comment]
@Marta108 Seems the Github does not replace the folder name Tic-tac-toe most likely you have to do it throw Github editor, we can have a look into this on Monday Also remember resolve conversations

**Pull request merged**